src/sample/hmr.ts

```diff
@@ -1,22 +1,11 @@
-import { applyPolyfill, reflowDOM, onCustomElementChange } from 'custom-elements-hmr-polyfill';
+import { applyPolyfill } from 'custom-elements-hmr-polyfill';
 
-// apply polly fill
-applyPolyfill();
+applyPolyfill(
+    /* enable autoReflow */ true,
+    /* buffer changes for 150ms */ 150,
+    /* called for every web component code change (ignores the buffer) */
 
-let awaiter: any;
-
-// listen for code changes
-onCustomElementChange((elementName: string, impl: any, options: ElementDefinitionOptions) => {
-    console.log('Detected a code change for custom element', elementName);
-
-    // most simple buffering algorithm, all reg. callbacks get cleared until one is the last one > 250ms
-    clearTimeout(awaiter);
-
-    awaiter = setTimeout(() => {
-        console.log('[Buffered reflow] Re-flowing DOM to activate updated custom elements code.');
-
-        // reflow the DOM to re-create all elements (thus replacing elements and execute the new code)
-        // TODO: This could be improved by only replacing nodes matching the elementName that changed
-        reflowDOM();
-    }, 250 /* essential buffer time*/);
-});
+    (elementName: string, impl: any, options: ElementDefinitionOptions) => {
+        console.log('[Web Component code change] ', elementName, impl, options);
```

Comment:
Maybe we could have 2 console.log messages here, where the one printing everything is commented out. One the enabled just says the name.
Its printing so much at that I had to disable it to better see whats happending